server.js

```diff
@@ -1,33 +1,97 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import netflixData from './data/netflix-titles.json'
+import { filterMoviesOnYear, filterMoviesByCountry, filterMoviesByActor, mappedNetflix } from './filters'
```

Comment:
GREAT!! Utility functions extracted to separate file makes the server much easier to maintain :)